lib/platform/gitea/gitea-helper.ts

```diff
@@ -135,7 +136,9 @@ export type RepoSearchParams = {
   archived?: boolean;
 };
 
-export type IssueCreateParams = IssueUpdateParams;
+export type IssueCreateParams = {
+  labels?: number[];
+} & IssueUpdateParams;
```

Comment:
```suggestion
export type IssueCreateParams = Partial<IssueUpdateLabelsParams> & IssueUpdateParams;
```

I think this looks better. 🙃

Reply:
I agree :)